.github/workflows/release.yml

```diff
@@ -30,13 +37,13 @@ jobs:
       - uses: nearform-actions/optic-release-automation-action@v4
         with:
           commit-message: 'Release {version}'
-          sync-semver-tags: true
+          sync-semver-tags: false
           access: 'public'
           # This prefix is added before the prerelease number, e.g. `v3.0.0-next.0`
-          prerelease-prefix: 'next'
+          # prerelease-prefix: 'alpha'
           semver: ${{ github.event.inputs.semver }}
-          # Prereleases are published under the `next` npm dist-tag
-          npm-tag: ${{ startsWith(github.event.inputs.semver, 'pre') && 'next' || 'latest' }}
+          # Prereleases are published under the `alpha` npm dist-tag
+          # npm-tag: ${{ startsWith(github.event.inputs.semver, 'pre') && 'alpha' || 'latest' }}
```

Comment:
we haven't made use of pre-releases for this module yet so figured it's safe to revert/comment out these inputs